src/cf-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-core",
-  "version": "4.7.0",
+  "version": "4.8.0",
```

Comment:
@contolini 

![whatd_you_do_tommy_boy](https://user-images.githubusercontent.com/1280430/44425060-74c78680-a550-11e8-9261-1f8b6a2cb81e.gif)

Reply:
Don't ask questions you don't want to know the answer to ;)

Reply:
😆 I saw the rest of the conversation, never would have guessed it was Jest. Nice sleuthing on @contolini's part